src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -502,18 +553,6 @@ class Diagram extends Component<Props> {
             <MenuItem text={flow.workflow} onClick={wrap(this.add.gotoSubWorkflow, point, flow.workflowPath)} />
           ))}
         </MenuItem>
-
-        <MenuItem tagName="span" text={lang.tr('skills')} icon="add">
-          {this.props.skills.map(skill => (
-            <MenuItem
-              key={skill.id}
-              text={lang.tr(skill.name)}
-              tagName="button"
-              onClick={wrap(this.add.skillNode, point, skill.id)}
-              icon={skill.icon}
-            />
-          ))}
-        </MenuItem>
```

Comment:
@EFF Do we really want to remove this while we have no replacement? (eg: send email, call api). I know the goal is to make them as a reusable, but I'd deprecate them before removing them...

Reply:
let's remove them and implement them as re-usable. No way we ship LTS without that